### PR TITLE
docs: add working group details

### PR DESCRIPTION
**Pull request by UlisesGascon**
This is an initial version, feel free to add you feedback.

There are some areas with `_TBC_` as I am working in parallel with the teams initial composition, I can push the changes to this PR or in a separate PR if this one gets merged first.

### CODE REVIEW

**Comment by UlisesGascon** [comment]
This is ready for review (cc: @expressjs/express-tc)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,63 @@
...
+
+## Charter
+
+The Security Working Group manages all aspects and processes linked to Express.js security, and is responsible for managing incoming security reports, and responsible also to prepare patches or releases. The nature of this task is sensitive, so only the Security triage team, Repo Captains and TC members will be involved on it.
+
+**Responsibilities include**
+- Define the Security triage role
+- Define and maintain security policies and procedures for the project and the packages in scope
+- Elaborate guidelines and recommendations for the ecosystem on how to build more secure middleware
+- Review and recommend processes for handling of security reports (but not the actual handling of security reports, which are reviewed directly by the TC).
```
Comment (marco-ippolito) [inline README.md:11]:
```suggestion
- Review and recommend processes for handling of security reports (but not the actual handling of security reports, which are reviewed directly by the TC).
```

```suggestion
- Review and recommend processes for handling of security reports (but not the actual handling of security reports, which are reviewed directly by the TC).
```

This look confusing, why triagers are not handling security reports?

Comment (UlisesGascon) [reply README.md]: Yes @marco-ippolito, seems like I made a mistake when I was copying it. So, amend the commit.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,63 @@
...
+- [Íñigo Marquínez Prado](https://github.com/inigomarquinez)
+- [Jean Burellier](https://github.com/sheplu)
+- [Marco Ippolito](https://github.com/marco-ippolito)
+- [Rafael Gonzaga](https://github.com/RafaelGSS)
+- [Ulises Gascón](https://github.com/UlisesGascon)
+- [Wes Todd](https://github.com/wesleytodd)
+
+## Meetings
+
+The Security Working Group meets every two weeks. Meetings are held on Zoom and are recorded or directly streamed to Youtube. The meeting is open to the public. The agenda and meeting notes are published in this repository.
```
Comment (inigomarquinez) [inline README.md:55]: Does it make sense to add a reference to the meetings calendar [here](https://openjsf.org/collaboration)? Or these meetings are not published in that calendar?

**Review by inigomarquinez — COMMENTED** [review]
A small question/suggestion added regarding the meetings section.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,63 @@
-# security-wg
-Express.js Security Working Group
+# Security Working Group
+
+## Charter
+
+The Security Working Group manages all aspects and processes linked to Express.js security, and is responsible for managing incoming security reports, and responsible also to prepare patches or releases. The nature of this task is sensitive, so only the Security triage team, Repo Captains and TC members will be involved on it.
```
Comment (wesleytodd) [inline README.md:5]: I made a similar edit in the other doc. There is some ambiguity with this, but I think the project is not named `Express.js` and I think that this group has responsibilities beyond the `express` package itself. We should probably say "the Express project" everywhere as a way to indicate the entire collection of packages.

```suggestion
The Security Working Group manages all aspects and processes linked to the Express project's security, and is responsible for managing incoming security reports, and responsible also to prepare patches or releases. The nature of this task is sensitive, so only the Security triage team, Repo Captains and TC members will be involved on it.
```

File: README.md (hunk 2 above)
Comment (wesleytodd) [reply README.md:55]: The are likely not yet, but I think this is a good idea and we can link the calendar even before they are added to it.

**Review by wesleytodd — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,2 +1,63 @@
...
+
+**Responsibilities include**
```
Comment (ctcpip) [inline README.md:7]:
```suggestion
### Responsibilities

```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,63 @@
...
+- Define the Security triage role
+- Define and maintain security policies and procedures for the project and the packages in scope
+- Elaborate guidelines and recommendations for the ecosystem on how to build more secure middleware
```
Comment (ctcpip) [inline README.md:10]:
```suggestion
- Provide guidance to the ecosystem on how to build more secure middleware
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,63 @@
...
+- Review and recommend processes for handling of security reports.
+- Promote improvement of security practices within the Express.js ecosystem (For example: [OSSF Scorecard](https://github.com/expressjs/discussions/issues/162), threat model, etc..)
```
Comment (ctcpip) [inline README.md:12]: per @wesleytodd's comment, update express.js to match any change

File: README.md [hunk 7]
```diff
@@ -1,2 +1,63 @@
...
+- Recommend security improvements for the project and the packages in scope
+- Support the TC team on security triage when is requested
```
Comment (ctcpip) [inline README.md:14]:
```suggestion
- Support the TC team on security triage as needed
```

File: README.md [hunk 8]
```diff
@@ -1,2 +1,63 @@
...
+- Support initiatives from the [OpenJS Foundation Security Collab Space](https://github.com/openjs-foundation/security-collab-space).
+
```
Comment (ctcpip) [inline README.md:16]:
```suggestion
- Support initiatives from the OpenSSF [Best Practices for Open Source Developers Working Group](https://github.com/ossf/wg-best-practices-os-developers).
```

even if I'm the only one who participates there, perhaps this is an easy addition here?

File: README.md [hunk 9]
```diff
@@ -1,2 +1,63 @@
...
+
+## Current Initiatives
+
+We are currently defining the Initiatives for 2024, [feel free to participate](https://github.com/expressjs/security-wg/issues/1).
+
+| Initiative | Champion | Status | Links |
+|------------|----------|--------|-------|
+| OSSF Scorecard | [@inigomarquinez](https://github.com/inigomarquinez) | In progress | [#2](https://github.com/expressjs/security-wg/issues/2)|
```
Comment (ctcpip) [inline README.md:24]: I am loathe to pile on to the large list of things we want to get done, but we should consider configuring [allstar](https://github.com/ossf/allstar#what-is-allstar) at some point as well

File: README.md [hunk 10]
```diff
@@ -1,2 +1,63 @@
...
+| Threat Model | _TBC_ | In progress | [#3](https://github.com/expressjs/security-wg/issues/3) |
+| Support OSTIF Audit | [@UlisesGascon](https://github.com/ulisesgascon) | In progress | [#6](https://github.com/expressjs/security-wg/issues/6)
+
+## Members
+
+The Security Working Group is composed by two groups of members: the Security Triage Team and the Regular members. The regular members are responsible for the public facing activity of the group, while the Security Triage Team is responsible for the security triage process.
```
Comment (ctcpip) [inline README.md:30]:
```suggestion
The Security Working Group is composed of two groups of members: the Security Triage Team and the Regular members. The regular members are responsible for the public facing activity of the group, while the Security Triage Team is responsible for the security triage process.
```

File: README.md [hunk 11]
```diff
@@ -1,2 +1,63 @@
...
+- Recommend security improvements for the project and the packages in scope
+- Support the TC team on security triage when is requested
+- Support initiatives from the [OpenJS Foundation Security Collab Space](https://github.com/openjs-foundation/security-collab-space).
+
+
+## Current Initiatives
+
+We are currently defining the Initiatives for 2024, [feel free to participate](https://github.com/expressjs/security-wg/issues/1).
+
+| Initiative | Champion | Status | Links |
```
[the remark this answers is not in the document]
Comment (UlisesGascon) [reply README.md:22]: Yes 🙏

File: README.md (hunk 8 above)
Comment (UlisesGascon) [reply README.md]: I love the idea, I added a comment in #1

File: README.md (hunk 9 above)
Comment (UlisesGascon) [reply README.md:24]: Great Idea! Added to the list https://github.com/expressjs/security-wg/issues/1

File: README.md [hunk 12]
```diff
@@ -1,2 +1,64 @@
...
+
+The Security Working Group meets every two weeks. Meetings are held on Zoom and are recorded or directly streamed to Youtube. The meeting is open to the public. The agenda and meeting notes are published in this repository.
+
+## Offline Discussions
+
+The Security Working Group uses the [GitHub issues](https://github.com/expressjs/security-wg/issues) for offline discussions. The discussions are open to the public and anyone can participate. Also, the group uses the channel `#express-security-wg` in the [OpenJS Foundation Slack](https://openjsf.org/collaboration) for real-time discussions.
+
+## Code of Conduct
+
+The  [Express.js CoC](https://github.com/expressjs/express/blob/master/Code-Of-Conduct.md) applies to this repo.
```
Comment (ctcpip) [inline README.md:64]: may need up for consistency (express.js)

File: README.md (hunk 2 above)
Comment (UlisesGascon) [reply README.md]: Thanks for the suggestion! I added it in the commit d1c993b12263c66f0674d8f020dc6c918dfbb88b

File: README.md (hunk 3 above)
Comment (UlisesGascon) [reply README.md]: Updated in b5a4abfc80f911bd7e41cba8b2aa8e8a6e668b8b

File: README.md (hunk 6 above)
Comment (UlisesGascon) [reply README.md]: Updated in b5a4abfc80f911bd7e41cba8b2aa8e8a6e668b8b

File: README.md (hunk 12 above)
Comment (UlisesGascon) [reply README.md]: Solved at 317ff29dbb6a00c815bce0c43f24d0954c442ef8

**Comment by UlisesGascon** [comment]
I think that this can be merged with an additional approval 👍

**Review by marco-ippolito — APPROVED** [review]

**Pull request merged**